source/components/molecules/Modal/Modal.js

```diff
@@ -1,33 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import RnModal from 'react-native-modal';
-import styled from 'styled-components/native';
-import { SafeAreaView } from 'react-native-safe-area-context';
+import { Modal as RnModal } from 'react-native';
 
-const ModalContainer = styled(RnModal)`
-  margin-left: 0px;
-  margin-right: 0px;
-  margin-bottom: 0px;
-  margin-top: 0px;
-  border-top-left-radius: 17.5px;
-  border-top-right-radius: 17.5px;
-`;
-
-const Modal = ({ visible, children, ...other }) => (
-  <ModalContainer
-    animationInTiming={500}
-    animationOutTiming={500}
-    propagateSwipe
-    isVisible={visible}
-    transparent
+const Modal = ({ visible, hide, children, ...other }) => (
```

Comment:
what is other?

Reply:
This way of destructing is a react practice, this makes it possible to pass props that is not declared in this component down. As you can see we pass this other value to the RnModal, this is done if there are props on the RnModal that we want to pass but not declare in this component explicitly :)